tools/windows/install-help/cal/ddreg.cpp

```diff
@@ -104,9 +104,9 @@ bool RegKey::getDWORDValue(const wchar_t *valname, DWORD &val)
 
 bool RegKey::setDWORDValue(const wchar_t *valname, DWORD value)
 {
-    RegSetValueEx(this->hKeyRoot, valname, 0, REG_DWORD, (const BYTE *)&value,
-                  (DWORD)sizeof(DWORD));
-    return true;
+    return RegSetValueEx(this->hKeyRoot, valname, 0, REG_DWORD, (const BYTE *)&value,
+                  (DWORD)sizeof(DWORD)) == ERROR_SUCCESS ? true : false;
+    
```

Comment:
I think "? true : false" is redundant. To me though it is readable as well as without it hence I suggest to change if you agree and it is not too much to bother, but it is up-to-you

Reply:
@iglendd I'm going to pass on changing this in this PR.
While I agree that it is redundant, this pattern exists elsewhere in the code.  If I change it here, I should change it all of the other places, which just adds more churn to the PR.